lib/turbo_tests.rb

```diff
@@ -2,7 +2,6 @@
 
 require "open3"
 require "fileutils"
-require "ostruct"
 require "json"
```

Comment:
`json` gem is used [`TurboTests::JsonRowsFormatter#JsonRowsFormatter`](https://github.com/serpapi/turbo_tests/blob/703121054c785a364bf4ab499eeb72868e61bf68/lib/turbo_tests/json_rows_formatter.rb#L167) to provide `to_json` method, similarly to the [`RSpec::Core::Formatters::JsonFormatter#close`](https://github.com/rspec/rspec-core/blob/1e661db5c5b431c0ee88a383e8e3767f02dccbfe/lib/rspec/core/formatters/json_formatter.rb#L56-L58) method.